packages/mdc-grid-list/mdc-grid-list.scss

```diff
@@ -0,0 +1,237 @@
+  @include mdc-grid-list--tile-aspect(2 / 3);
+}
+
+.mdc-grid-list--tile-aspect-4x3 {
+  @include mdc-grid-list--tile-aspect(4 / 3);
+}
+
+.mdc-grid-list--tile-aspect-3x4 {
+  @include mdc-grid-list--tile-aspect(3 / 4);
+}
+
+.mdc-grid-list--twoline-caption {
+  .mdc-grid-tile__secondary {
+    height: $mdc-grid-list-twoline-cap-secondary-height;
+  }
+}
+
+.mdc-grid-list--header-caption {
+  .mdc-grid-tile__secondary {
+    top: 0;
+    bottom: auto;
+  }
+}
+
+.mdc-grid-list--with-icon-align-start {
+  .mdc-grid-tile__secondary {
+    padding-right: $mdc-grid-list-tile-secondary-padding-narrow;
+    padding-left: $mdc-grid-list-tile-secondary-padding * 2 + $mdc-grid-list-tile-secondary-icon-size;
+
+    .mdc-grid-tile__icon {
+      left: $mdc-grid-list-tile-secondary-padding;
+      font-size: $mdc-grid-list-tile-secondary-icon-size;
+    }
+  }
+
+  @include mdc-rtl {
+    .mdc-grid-tile__secondary {
+      padding-right: $mdc-grid-list-tile-secondary-padding * 2 + $mdc-grid-list-tile-secondary-icon-size;
+      padding-left: $mdc-grid-list-tile-secondary-padding-narrow;
+
+      .mdc-grid-tile__icon {
+        right: $mdc-grid-list-tile-secondary-padding;
+      }
+    }
+  }
+}
+
+.mdc-grid-list--with-icon-align-end {
+  .mdc-grid-tile__secondary {
+    padding-right: $mdc-grid-list-tile-secondary-padding * 2 + $mdc-grid-list-tile-secondary-icon-size;
+
+    .mdc-grid-tile__icon {
+      right: $mdc-grid-list-tile-secondary-padding;
+      font-size: $mdc-grid-list-tile-secondary-icon-size;
+    }
+  }
+
+  @include mdc-rtl {
+    .mdc-grid-tile__secondary {
+      padding-left: $mdc-grid-list-tile-secondary-padding * 2 + $mdc-grid-list-tile-secondary-icon-size;
+
+      .mdc-grid-tile__icon {
+        left: $mdc-grid-list-tile-secondary-padding;
+      }
+    }
+  }
+}
+// stylelint-enable plugin/selector-bem-pattern
+// postcss-bem-linter: end
+
+// postcss-bem-linter: define grid-tile
+.mdc-grid-tile {
+  // stylelint-disable plugin/selector-bem-pattern
+  --mdc-grid-tile-width: 200px;
+  // stylelint-enable plugin/selector-bem-pattern
+
+  display: block;
+  position: relative;
+  width: var(--mdc-grid-list-tile-width);
+  width: $mdc-grid-list-tile-width;
+
+  &__primary {
+    position: relative;
+    height: 0;
+
+    @include mdc-theme-prop(background-color, background);
+    @include mdc-theme-prop(color, text-primary-on-background);
+
+    &-content {
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      background-repeat: no-repeat;
+      background-position: center;
+      background-size: cover;
+    }
+  }
+
+  &__secondary {
+    position: absolute;
+    box-sizing: border-box;
+
+    @include mdc-theme-prop(background-color, primary);
+    @include mdc-theme-prop(color, text-primary-on-primary);
+
+    .mdc-grid-tile__title {
+      display: block;
+      margin: 0;
+      padding: 0;
+      font-size: 16px;
```

Comment:
for font styles, we almost always use `mdc-typography`, so I would use that if it makes sense. Otherwise, ensure that you always use `em` over `px` for fonts, as `em`s adjust to the user's default font size